poc-data-api/Extracting_Water_Temperature_at_Site.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Extracting temperature at specific reefs"
+author: "Denisse Fierro Arcos"
+date: "2023-06-05"
+output: 
+  github_document:
+    toc: true
+    html_preview: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Goal of this notebook
+
+This notebook will demonstrate how to access the RIMReP `geoparquet` collection for AIMS Sea Surface Temperature Monitoring Program. This dataset includes data since 1980 for tropical and subtropical coral reefs around Australia, including 80 Great Barrier Reef sites.
+
+Then, we will extract the coordinates for all sites sampled in this monitoring program. This way we can extract data for the site of our choice using the name of the site of our interest, without the need to know their coordinates.
+
+# Loading libraries
+
+```{r libraries, results = "hide", message = F, warning = F}
+library(arrow)
+library(dplyr)
+library(magrittr)
+library(stringr)
+library(wkb)
+library(ggplot2)
+library(sf)
+library(lubridate)
+```
+
+# Connecting to RIMReP collection
+
+As mentioned above, we will connect to the AIMS Sea Surface Temperature Monitoring Program. This can take a minute or so.
+
+```{r connect}
+#Establishing connection
+data_bucket <- s3_bucket("s3://rimrep-data-public/091-aims-sst/test-50-64-spatialpart")

```

Comment:
Is this the correct bucket? Check with Leo

Reply:
I checked in STAC and it shows the same URL.